business/istio_status.go

```diff
@@ -198,6 +202,14 @@ func (iss *IstioStatusService) getStatusOf(ds []apps_v1.Deployment) (IstioCompon
 		}
 	}
 
+	// When all the deployments are missing,
+	// Warn users that their kiali config might be wrong
+	if componentNotFound == len(statusComponents) {
+		return isc, errors.New(fmt.Sprintf(
+			"Are you sure Istio is installed in the %s namespace?",
```

Comment:
This message may lead a user to think that Istio *should be* installed in the %s namespace.    I think we are trying to warn them that this is where we are unsuccessfully looking for Istio.  Maybe something like this would be more clear?:

`Kiali is unable to find any Istio deployment in namespace %s.  Are you sure the Istio namespace is configured correctly in Kiali?`